src/App/App.js

```diff
@@ -30,27 +30,29 @@ class App extends Component {
       <article className='App'>
         <h1>Tainted Tomatillos</h1>
         <h2 className='small-title'>Where The Ratings Are Honestly Rotten</h2>
+        {this.state.error && <h3>{this.state.error}</h3>}
```

Comment:
I'm so glad you got this. I was close but I hadn't wrapped it in tags! This was something I wanted to talk to you about, so I'm glad you got it